test/lookup.cpp

```diff
@@ -127,7 +134,13 @@ class MyHandler : public DNS::Handler
                 case ns_t_ns:       std::cout << DNS::NS(response, record).nameserver(); break;
                 case ns_t_ptr:      std::cout << DNS::PTR(response, record).target(); break;
                 case ns_t_soa:      { DNS::SOA soa(response, record); std::cout << soa.nameserver() << " " << soa.email() << " " << soa.serial() << " " << soa.interval() << " " << soa.retry() << " " << soa.expire() << " " << soa.minimum(); } break;
-    //            case ns_t_caa:      std::cout << DNS::CAA(response, record); break;
+                case ns_t_tlsa:
+                {
+                        const DNS::TLSA tlsa(response, record);
+                        std::cout << (int)tlsa.certificateUsage() << " " << (int)tlsa.selector() << " " << (int)tlsa.matchingType() << " ";
+                        printhex(std::cout, tlsa.certificateAssociationData(), tlsa.certificateAssociationDataSize());
+                        break;
+                }
```

Comment:
I'm missing the processing of the security, which is actually needed for TLSA records I belive

Reply:
Let's for now focus on getting a record type added. The actual security verification could be done in a later step.